templates/ui/package.json

```diff
@@ -8,11 +8,11 @@
     "lint": "eslint src",
     "format": "format src"
   },
-  "devDependencies": {    
+  "devDependencies": {
     "@hcengineering/platform-rig": "^0.6.0",
-    "svelte-loader":"^3.1.3",
+    "svelte-loader":"^3.1.9",
     "sass":"^1.53.0",
-    "svelte-preprocess":"5.0.3",
+    "svelte-preprocess":"^5.1.0",
```

Comment:
You need to apply to all plugins after changing this file, so let's make it as separate PR.

Reply:
I've just checked. _All_ plugins do already depend on `^5.1.0`

Reply:
✅